pkg/splunksdk/pkg/utils/helper.go

```diff
@@ -16,64 +16,68 @@ const GetTriggeredInstances = "getTriggeredInstances"
 
 // mock an http server
 func MockRequest(response string, sslVerificationActivated bool) *httptest.Server {
-	server := &httptest.Server{}
-	if sslVerificationActivated {
+	var server *httptest.Server
+	switch sslVerificationActivated {
+	case true:
 		server = httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(200)
-			writeResponses(response, &w, r)
+			writeResponses(response, w, r)
 		}))
 
-	} else {
+	default:
 		server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(200)
-			writeResponses(response, &w, r)
+			writeResponses(response, w, r)
 		}))
 	}
 	return server
 }
 
 func MultitpleMockRequest(responses []map[string]interface{}, sslVerificationActivated bool) *httptest.Server {
-	server := &httptest.Server{}
-	if sslVerificationActivated {
+	var server *httptest.Server
+	switch sslVerificationActivated {
+	case true:
 		server = httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(200)
-			writeResponses(responses, &w, r)
+			writeResponses(responses, w, r)
 		}))
-	} else {
+	default:
 		server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.WriteHeader(200)
-			writeResponses(responses, &w, r)
+			writeResponses(responses, w, r)
 		}))
 	}
 	return server
 }
 
-func writeResponses(responses interface{}, w *http.ResponseWriter, r *http.Request) {
+func writeResponses(responses interface{}, w http.ResponseWriter, r *http.Request) {
 
 	switch resps := responses.(type) {
 	case []map[string]interface{}:
 		for _, response := range resps {
-			if r.Method == "GET" {
-				if response[GetTriggeredAlerts] != nil && strings.HasSuffix(r.URL.Path, "services/alerts/fired_alerts/") {
-					_, _ = (*w).Write([]byte(response[GetTriggeredAlerts].(string)))
-				} else if response[GetTriggeredInstances] != nil && strings.Contains(r.URL.Path, "search/alerts/fired_alerts/") {
-					_, _ = (*w).Write([]byte(response[GetTriggeredInstances].(string)))
-				} else if response[GetAlertsNames] != nil && strings.Contains(r.URL.Path, "services/saved/searches/") {
-					_, _ = (*w).Write([]byte(response[GetAlertsNames].(string)))
-				} else if response["GET"] != nil && r.Method == "GET" {
-					_, _ = (*w).Write([]byte(response["GET"].(string)))
+			switch method := r.Method; method {
+			case "GET":
```

Comment:
Not vital but you can use `http.MethodGet` here and `http.MethodPost` in line 70 instead of the hardcoded values